src/core/searcher.rs

```diff
@@ -23,6 +74,7 @@ pub struct Searcher {
     index: Index,
     segment_readers: Vec<SegmentReader>,
     store_readers: Vec<StoreReader>,
+    index_generation: TrackedObject<SearcherGeneration>,
```

Comment:
can name this just `generation` now perhaps

Reply:
might be worth grepping for usage of `index_generation` and considering a rename

Reply:
Grep for dexGen and dex_gen and removed occurences.